### DISCUSSION TITLE
Is MacOS supported for firox-sound standalone?

### DISCUSSION THREAD

**Discussion by John-Nagle** [FyroxEngine/Fyrox#625] [General]
I'm looking at using the [firox-sound crate](https://crates.io/crates/fyrox-sound) without any of the graphics components. Does that work? How's the cross-platform support?

Crates.io says:

> Supported OS
> 
>     Windows (DirectSound)
>     Linux (alsa)
>     macOS (CoreAudio)
>     WebAssembly (WebAudio)

but [docs.rs](https://docs.rs/fyrox-sound/0.34.0/fyrox_sound/) says.

> Supported OS:  Currently only Windows and Linux are supported.

What's the real situation? Thanks.

**Comment by mrDIMAS** [comment]
Docs are lying, crates.io telling the truth. I just forgot to change this line. 

Edit. It is fixed now (https://github.com/FyroxEngine/Fyrox/commit/321690264460861deae05f3ac326f960e2a1c08f).
